src/installer.ts

```diff
@@ -26,20 +26,36 @@ interface INodeVersionInfo {
 export async function getNode(
   versionSpec: string,
   stable: boolean,
+  checkLatest: boolean,
   auth: string | undefined
 ) {
   let osPlat: string = os.platform();
   let osArch: string = translateArchToDistUrl(os.arch());
 
+  if (checkLatest) {
+    core.info('Attempt to resolve the latest version from manifest...');
+    const resolvedVersion = await resolveVersionFromManifest(
+      versionSpec,
+      stable,
+      auth
+    );
+    if (resolvedVersion) {
+      versionSpec = resolvedVersion;
```

Comment:
cool.  so just to make 100% sure, if checkLatest, we resolve the inputted version spec against the node-versions cache manifest, and then rewrite the versionSpec as the resolved version.  I like that.